release/ray_release/util.py

```diff
@@ -148,8 +148,6 @@ def run_bash_script(bash_script: str) -> None:
 
 def reinstall_anyscale_dependencies() -> None:
     logger.info("Re-installing `anyscale` package")
-
-    # Copy anyscale pin to requirements.txt and requirements_buildkite.txt
```

Comment:
I do not understand why this comment is "copying anyscale pin". It seems that this is just reinstalling "anyscale" CLI.

Reply:
I think this comment means that you, the developer, should copy the pin to ensure consistency between this file and those other 2.

Reply:
ok. after this change there is no need to do that, because the dependency versions will all be pinned